packages/shared/src/validation/addressValidation.ts

```diff
@@ -1,5 +1,5 @@
 import * as bitcoin from '@chainflip/bitcoin';
-import { isValidSolanaAddress } from '@chainflip/solana/address';
+import { isValidSolanaAddress } from '@chainflip/solana';
```

Comment:
changed bc of https://github.com/chainflip-io/chainflip-product-toolkit/pull/304